bats/fb-content-katello.bats

```diff
@@ -121,8 +121,13 @@ EOF
     rpm -e `rpm -qf /etc/rhsm/ca/candlepin-local.pem`
   fi
 
+  yum erase -y 'katello-ca-consumer-*' || true
   rpm -Uvh http://localhost/pub/katello-ca-consumer-latest.noarch.rpm || true
   subscription-manager register --force --org="Test_Organization" --activationkey="Test AK" || true
+  if tIsRHEL && ! tIsCentOSCompatible; then
+    # skipping the rest of the test as we don't provide RHEL subs
+    skip
+  fi
```

Comment:
Yupp, just checked, we can do `subscription-manager list --consumed | grep "Test Product"`, what do you think?